### PR TITLE
Power-down fixes for vanilla ESP32

### PR DESCRIPTION
**Pull request by poesel**
fixes for HX711 & I2C for powering down on the esp32

### CODE REVIEW

File: terkin/lib/hx711.py [hunk 1]
```diff
@@ -235,7 +235,8 @@ def power_down(self):
 
         # Hold level to HIGH, even during deep sleep.
         # https://community.hiveeyes.org/t/strom-sparen-beim-einsatz-der-micropython-firmware-im-batteriebetrieb/2055/72
-        self.pSCK.hold(True)
+        if platform_info.vendor == platform_info.MICROPYTHON.Pycom:
+            self.pSCK.hold(True)
```
Comment (amotl) [inline terkin/lib/hx711.py]: How would we implement that for Vanilla ESP32?

File: terkin/sensor/core.py [hunk 2]
```diff
@@ -386,8 +386,9 @@ def power_on(self):
             self.just_started = False
             return
 
-        # TODO: Add implementation for Vanilla MicroPython.
-        self.adapter.init(mode=I2C.MASTER, baudrate=self.frequency)
+        # uPy doesn't have deinit so it doesn't need init
+        if platform_info.vendor == platform_info.MICROPYTHON.Pycom:
+            self.adapter.init(mode=I2C.MASTER, baudrate=self.frequency)
```
Comment (amotl) [inline terkin/sensor/core.py]: Isn't there something how this is done on Genuine MicroPython on Vanilla ESP32?

**Comment by amotl** [comment]
Thanks!

File: terkin/sensor/core.py (hunk 2 above)
Comment (poesel) [reply terkin/sensor/core.py]: uPys I2C has no/doesn't need init/deinit so AFAIK nothing can be done.

**Pull request merged**